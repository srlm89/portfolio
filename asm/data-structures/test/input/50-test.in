ERASE actual/50-test.out
INSERT 13 4.396584 Assassination of a High School President 720p
PRINT_IN_ORDER actual/50-test.out
INSERT 21 5.070408 007 Goldfinger [1964] [BluRay] [720p] [DTS]
INSERT 50 0.810797 Blood River 720P
REMOVE_ID 13
PRINT_IN_ORDER actual/50-test.out
INSERT 1 0.773411 Stargate-The Ark of Truth 720p
INSERT 15 9.423040 Underworld Series
INSERT 20 6.718305 Bronson.LIMITED.720p.BluRay.x264-iNFAMOUS
REMOVE_ID 15
PRINT_IN_ORDER actual/50-test.out
INSERT 9 8.702495 Spiderman 3   1080p
INSERT 37 5.064373 The Brave One 720P
INSERT 46 7.397625 Die Hard 4   1080p
INSERT 40 2.029920 Hulk 2 - The Incredible Hulk - 720p
REMOVE_ID 37
REMOVE_ID 21
PRINT_IN_ORDER actual/50-test.out
INSERT 39 1.234791 The Brave One 720P
INSERT 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p
INSERT 6 6.304932 Blade 2   720p
INSERT 41 5.175064 Robots.WS.AC3.x264.BluRay-PTM   720p
INSERT 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR
REMOVE_ID 6
REMOVE_ID 20
PRINT_IN_ORDER actual/50-test.out
INSERT 5 6.054702 Horseman    720p
INSERT 28 7.498223 Romeo And Juliet   720p
INSERT 2 7.597812 Dolphin    720p
INSERT 27 6.874840 Firewall 720p
INSERT 60 9.531191 Drag Me To Hell [720p]
INSERT 26 3.081259 Babel.720p
REMOVE_ID 26
REMOVE_ID 41
REMOVE_ID 50
PRINT_IN_ORDER actual/50-test.out
INSERT 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS
INSERT 59 2.255431 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p]
INSERT 38 4.523917 The Lion King [1994] [720p]
INSERT 54 9.232851 Con Air   720p
INSERT 19 5.958785 Eurotrip    720p
INSERT 7 2.022839 X-Men Origin - Wolverine [2009] [1080p]
INSERT 12 1.072862 Batman Cartoon - Gotham Knight   720p
REMOVE_ID 59
REMOVE_ID 46
REMOVE_ID 5
PRINT_IN_ORDER actual/50-test.out
INSERT 56 5.239891 Stargate-The Ark of Truth 720p
INSERT 76 1.010990 Terminator.1984.720p.BluRay.x264-ESiR
INSERT 18 1.933603 Facing The Giants 720p
INSERT 4 2.638189 enemy at the gates    1080p
INSERT 16 6.284310 Afro samurai uncut 720p
INSERT 61 6.838167 James Bond 01 - Dr. No [1962] [720p]
INSERT 10 6.844069 Crash    1080p
INSERT 30 6.731759 Juno 2007 720p
REMOVE_ID 9
REMOVE_ID 40
REMOVE_ID 16
REMOVE_ID 61
PRINT_IN_ORDER actual/50-test.out
INSERT 73 8.138925 Hero    1080p
INSERT 66 5.759800 Jumper    1080p
INSERT 47 6.025115 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20
INSERT 75 9.454583 E.T   720p
INSERT 49 1.966562 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR
INSERT 43 8.268652 Gran Torino [2008] [720p]
INSERT 44 0.000518 Ghost Rider [2007] [720p]
INSERT 0 4.239712 Memento    1080p
INSERT 94 0.054333 One Missed Call   720p
REMOVE_ID 73
REMOVE_ID 39
REMOVE_ID 43
REMOVE_ID 0
PRINT_IN_ORDER actual/50-test.out
INSERT 55 8.043650 Erin Brockovich 2000.720p
INSERT 57 6.004073 Leaving Las Vegas  720p
INSERT 48 6.816887 Fearless - 720p
INSERT 86 4.789126 Hitman    720p
INSERT 77 7.685619 1408.2007.720p.BluRay.x264-SiNNERS
INSERT 32 6.128344 James Bond 17 - Golden Eye [1995]
INSERT 82 8.131417 Charlie and the Chocolate Factory   720p
INSERT 74 3.851674 James Bond 01 - Dr. No [1962] [720p]
INSERT 93 7.309793 Pirates of Caribbean 1-2-3 Collection   1080p
INSERT 87 2.088651 007 Thunderball [1965]  [BluRay] [720p] [DTS]
REMOVE_ID 74
REMOVE_ID 1
REMOVE_ID 27
REMOVE_ID 47
REMOVE_ID 10
PRINT_IN_ORDER actual/50-test.out
INSERT 22 4.442938 This Is It [2009] [720p]
INSERT 98 2.822043 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON
INSERT 95 6.731524 Into The Blue 720P
INSERT 103 3.510903 Moulin Rouge   720p
INSERT 101 6.703355 The Secret of Moonarce [720p]
INSERT 78 7.596681 James Bond 15 - The Living Daylights [1987] [720p]
INSERT 88 7.326222 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON
INSERT 107 3.750715 The Notebook [2004] [1080p]
INSERT 108 9.545302 Fantastic Four, Rise Of The Silver Surfer    1080p
INSERT 62 1.311995 G.I.Joe The Rise of Cobra [2009] [720p]
INSERT 14 2.737409 SAW V
REMOVE_ID 38
REMOVE_ID 78
REMOVE_ID 103
REMOVE_ID 31
REMOVE_ID 60
PRINT_IN_ORDER actual/50-test.out
INSERT 65 1.848927 Punisher 2   720p
INSERT 24 2.418339 Eragon 720p
INSERT 58 4.944464 Incredibles, The 1080p
INSERT 90 0.025561 Surrogates [2009] [720p]
INSERT 112 1.821403 Dolphin    720p
INSERT 70 4.510654 Doomsday 720p
INSERT 83 7.210427 National Geographic - The Koran [720p] [Documenter]
INSERT 120 4.504557 Hancock    1080p
INSERT 104 6.991941 Core   720p
INSERT 123 6.170543 Babel.720p
INSERT 85 8.317092 12 Rounds 720p
INSERT 100 2.894432 Liar Liar    720p
REMOVE_ID 90
REMOVE_ID 112
REMOVE_ID 76
REMOVE_ID 77
REMOVE_ID 94
REMOVE_ID 75
PRINT_IN_ORDER actual/50-test.out
INSERT 118 7.655537 Alvin And The Chipmunks [720p]
INSERT 72 6.411729 SAW III   720p
INSERT 91 9.302142 Bangkok Dangerous 720p
INSERT 126 0.553977 Die Hard 3   720p
INSERT 81 7.386294 Kung Fu Panda [2008]   720p
INSERT 114 3.753160 Terminator 2 Judgement Day Directors Cut [1991]   1080p
INSERT 124 0.016130 X-Men 4 - Origins Wolverine (2009)  1080p
INSERT 79 1.444821 The Matrix   1080p
INSERT 36 3.873967 Crossing Over 720P
INSERT 80 6.262072 Dolphin    720p
INSERT 64 6.100061 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR
INSERT 84 6.077602 21   1080p
INSERT 69 7.068126 Before The Devil Knows, You're Dead   1080p
REMOVE_ID 24
REMOVE_ID 98
REMOVE_ID 108
REMOVE_ID 49
REMOVE_ID 87
REMOVE_ID 80
PRINT_IN_ORDER actual/50-test.out
INSERT 140 8.993602 The Proposal [2009] [720p]
INSERT 99 0.480487 One Missed Call   720p
INSERT 96 4.189030 Baby on Board (2009)    720p
INSERT 42 1.352024 Epic Movie UNRATED (2007) 720p
INSERT 111 3.670414 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs
INSERT 29 0.386847 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE
INSERT 138 4.200550 Crank 2, High Voltage   1080p
INSERT 23 4.753598 Eagle Eye    1080p
INSERT 141 5.388408 Dr Dolite 3    720p
INSERT 139 0.580774 Terminator 3 Rise Of The Machine [2003]   720p
INSERT 125 3.012549 catch me if u can - 720p
INSERT 105 2.985910 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC
INSERT 45 6.753766 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD
INSERT 136 9.956173 Godzilla 720p
REMOVE_ID 96
REMOVE_ID 28
REMOVE_ID 136
REMOVE_ID 12
REMOVE_ID 126
REMOVE_ID 139
REMOVE_ID 69
PRINT_IN_ORDER actual/50-test.out
