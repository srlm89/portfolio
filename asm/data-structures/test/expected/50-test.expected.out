[ 13 4.396584 Assassination of a High School President 720p ]
[ 21 5.070408 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 50 0.810797 Blood River 720P ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 20 6.718305 Bronson.LIMITED.720p.BluRay.x264-iNFAMOUS ][ 21 5.070408 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 50 0.810797 Blood River 720P ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 9 8.702495 Spiderman 3   1080p ][ 20 6.718305 Bronson.LIMITED.720p.BluRay.x264-iNFAMOUS ][ 40 2.029920 Hulk 2 - The Incredible Hulk - 720p ][ 46 7.397625 Die Hard 4   1080p ][ 50 0.810797 Blood River 720P ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 9 8.702495 Spiderman 3   1080p ][ 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 39 1.234791 The Brave One 720P ][ 40 2.029920 Hulk 2 - The Incredible Hulk - 720p ][ 41 5.175064 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 46 7.397625 Die Hard 4   1080p ][ 50 0.810797 Blood River 720P ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 2 7.597812 Dolphin    720p ][ 5 6.054702 Horseman    720p ][ 9 8.702495 Spiderman 3   1080p ][ 27 6.874840 Firewall 720p ][ 28 7.498223 Romeo And Juliet   720p ][ 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 39 1.234791 The Brave One 720P ][ 40 2.029920 Hulk 2 - The Incredible Hulk - 720p ][ 46 7.397625 Die Hard 4   1080p ][ 60 9.531191 Drag Me To Hell [720p] ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 9 8.702495 Spiderman 3   1080p ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 19 5.958785 Eurotrip    720p ][ 27 6.874840 Firewall 720p ][ 28 7.498223 Romeo And Juliet   720p ][ 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 38 4.523917 The Lion King [1994] [720p] ][ 39 1.234791 The Brave One 720P ][ 40 2.029920 Hulk 2 - The Incredible Hulk - 720p ][ 54 9.232851 Con Air   720p ][ 60 9.531191 Drag Me To Hell [720p] ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 10 6.844069 Crash    1080p ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 27 6.874840 Firewall 720p ][ 28 7.498223 Romeo And Juliet   720p ][ 30 6.731759 Juno 2007 720p ][ 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 38 4.523917 The Lion King [1994] [720p] ][ 39 1.234791 The Brave One 720P ][ 54 9.232851 Con Air   720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 60 9.531191 Drag Me To Hell [720p] ][ 76 1.010990 Terminator.1984.720p.BluRay.x264-ESiR ]
[ 1 0.773411 Stargate-The Ark of Truth 720p ][ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 10 6.844069 Crash    1080p ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 27 6.874840 Firewall 720p ][ 28 7.498223 Romeo And Juliet   720p ][ 30 6.731759 Juno 2007 720p ][ 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 38 4.523917 The Lion King [1994] [720p] ][ 44 0.000518 Ghost Rider [2007] [720p] ][ 47 6.025115 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 49 1.966562 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 54 9.232851 Con Air   720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 60 9.531191 Drag Me To Hell [720p] ][ 66 5.759800 Jumper    1080p ][ 75 9.454583 E.T   720p ][ 76 1.010990 Terminator.1984.720p.BluRay.x264-ESiR ][ 94 0.054333 One Missed Call   720p ]
[ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 28 7.498223 Romeo And Juliet   720p ][ 30 6.731759 Juno 2007 720p ][ 31 6.227451 The.Godfather.Part.I.1972.720p.BluRay.x264-ESiR ][ 32 6.128344 James Bond 17 - Golden Eye [1995] ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 38 4.523917 The Lion King [1994] [720p] ][ 44 0.000518 Ghost Rider [2007] [720p] ][ 48 6.816887 Fearless - 720p ][ 49 1.966562 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 54 9.232851 Con Air   720p ][ 55 8.043650 Erin Brockovich 2000.720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 57 6.004073 Leaving Las Vegas  720p ][ 60 9.531191 Drag Me To Hell [720p] ][ 66 5.759800 Jumper    1080p ][ 75 9.454583 E.T   720p ][ 76 1.010990 Terminator.1984.720p.BluRay.x264-ESiR ][ 77 7.685619 1408.2007.720p.BluRay.x264-SiNNERS ][ 82 8.131417 Charlie and the Chocolate Factory   720p ][ 86 4.789126 Hitman    720p ][ 87 2.088651 007 Thunderball [1965]  [BluRay] [720p] [DTS] ][ 93 7.309793 Pirates of Caribbean 1-2-3 Collection   1080p ][ 94 0.054333 One Missed Call   720p ]
[ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 14 2.737409 SAW V ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 22 4.442938 This Is It [2009] [720p] ][ 28 7.498223 Romeo And Juliet   720p ][ 30 6.731759 Juno 2007 720p ][ 32 6.128344 James Bond 17 - Golden Eye [1995] ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 44 0.000518 Ghost Rider [2007] [720p] ][ 48 6.816887 Fearless - 720p ][ 49 1.966562 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 54 9.232851 Con Air   720p ][ 55 8.043650 Erin Brockovich 2000.720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 57 6.004073 Leaving Las Vegas  720p ][ 62 1.311995 G.I.Joe The Rise of Cobra [2009] [720p] ][ 66 5.759800 Jumper    1080p ][ 75 9.454583 E.T   720p ][ 76 1.010990 Terminator.1984.720p.BluRay.x264-ESiR ][ 77 7.685619 1408.2007.720p.BluRay.x264-SiNNERS ][ 82 8.131417 Charlie and the Chocolate Factory   720p ][ 86 4.789126 Hitman    720p ][ 87 2.088651 007 Thunderball [1965]  [BluRay] [720p] [DTS] ][ 88 7.326222 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 93 7.309793 Pirates of Caribbean 1-2-3 Collection   1080p ][ 94 0.054333 One Missed Call   720p ][ 95 6.731524 Into The Blue 720P ][ 98 2.822043 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 101 6.703355 The Secret of Moonarce [720p] ][ 107 3.750715 The Notebook [2004] [1080p] ][ 108 9.545302 Fantastic Four, Rise Of The Silver Surfer    1080p ]
[ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 14 2.737409 SAW V ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 22 4.442938 This Is It [2009] [720p] ][ 24 2.418339 Eragon 720p ][ 28 7.498223 Romeo And Juliet   720p ][ 30 6.731759 Juno 2007 720p ][ 32 6.128344 James Bond 17 - Golden Eye [1995] ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 44 0.000518 Ghost Rider [2007] [720p] ][ 48 6.816887 Fearless - 720p ][ 49 1.966562 Lucky.Number.Slevin.2006.720p.HDDVD.x264-ESiR ][ 54 9.232851 Con Air   720p ][ 55 8.043650 Erin Brockovich 2000.720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 57 6.004073 Leaving Las Vegas  720p ][ 58 4.944464 Incredibles, The 1080p ][ 62 1.311995 G.I.Joe The Rise of Cobra [2009] [720p] ][ 65 1.848927 Punisher 2   720p ][ 66 5.759800 Jumper    1080p ][ 70 4.510654 Doomsday 720p ][ 82 8.131417 Charlie and the Chocolate Factory   720p ][ 83 7.210427 National Geographic - The Koran [720p] [Documenter] ][ 85 8.317092 12 Rounds 720p ][ 86 4.789126 Hitman    720p ][ 87 2.088651 007 Thunderball [1965]  [BluRay] [720p] [DTS] ][ 88 7.326222 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 93 7.309793 Pirates of Caribbean 1-2-3 Collection   1080p ][ 95 6.731524 Into The Blue 720P ][ 98 2.822043 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 100 2.894432 Liar Liar    720p ][ 101 6.703355 The Secret of Moonarce [720p] ][ 104 6.991941 Core   720p ][ 107 3.750715 The Notebook [2004] [1080p] ][ 108 9.545302 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 120 4.504557 Hancock    1080p ][ 123 6.170543 Babel.720p ]
[ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 12 1.072862 Batman Cartoon - Gotham Knight   720p ][ 14 2.737409 SAW V ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 22 4.442938 This Is It [2009] [720p] ][ 28 7.498223 Romeo And Juliet   720p ][ 30 6.731759 Juno 2007 720p ][ 32 6.128344 James Bond 17 - Golden Eye [1995] ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 36 3.873967 Crossing Over 720P ][ 44 0.000518 Ghost Rider [2007] [720p] ][ 48 6.816887 Fearless - 720p ][ 54 9.232851 Con Air   720p ][ 55 8.043650 Erin Brockovich 2000.720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 57 6.004073 Leaving Las Vegas  720p ][ 58 4.944464 Incredibles, The 1080p ][ 62 1.311995 G.I.Joe The Rise of Cobra [2009] [720p] ][ 64 6.100061 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 65 1.848927 Punisher 2   720p ][ 66 5.759800 Jumper    1080p ][ 69 7.068126 Before The Devil Knows, You're Dead   1080p ][ 70 4.510654 Doomsday 720p ][ 72 6.411729 SAW III   720p ][ 79 1.444821 The Matrix   1080p ][ 81 7.386294 Kung Fu Panda [2008]   720p ][ 82 8.131417 Charlie and the Chocolate Factory   720p ][ 83 7.210427 National Geographic - The Koran [720p] [Documenter] ][ 84 6.077602 21   1080p ][ 85 8.317092 12 Rounds 720p ][ 86 4.789126 Hitman    720p ][ 88 7.326222 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 91 9.302142 Bangkok Dangerous 720p ][ 93 7.309793 Pirates of Caribbean 1-2-3 Collection   1080p ][ 95 6.731524 Into The Blue 720P ][ 100 2.894432 Liar Liar    720p ][ 101 6.703355 The Secret of Moonarce [720p] ][ 104 6.991941 Core   720p ][ 107 3.750715 The Notebook [2004] [1080p] ][ 114 3.753160 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 118 7.655537 Alvin And The Chipmunks [720p] ][ 120 4.504557 Hancock    1080p ][ 123 6.170543 Babel.720p ][ 124 0.016130 X-Men 4 - Origins Wolverine (2009)  1080p ][ 126 0.553977 Die Hard 3   720p ]
[ 2 7.597812 Dolphin    720p ][ 3 9.975537 Oceans.Eleven.2001.720p.HDDVD.x264-SiNNERS ][ 4 2.638189 enemy at the gates    1080p ][ 7 2.022839 X-Men Origin - Wolverine [2009] [1080p] ][ 14 2.737409 SAW V ][ 18 1.933603 Facing The Giants 720p ][ 19 5.958785 Eurotrip    720p ][ 22 4.442938 This Is It [2009] [720p] ][ 23 4.753598 Eagle Eye    1080p ][ 29 0.386847 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 30 6.731759 Juno 2007 720p ][ 32 6.128344 James Bond 17 - Golden Eye [1995] ][ 35 7.290227 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 36 3.873967 Crossing Over 720P ][ 42 1.352024 Epic Movie UNRATED (2007) 720p ][ 44 0.000518 Ghost Rider [2007] [720p] ][ 45 6.753766 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 48 6.816887 Fearless - 720p ][ 54 9.232851 Con Air   720p ][ 55 8.043650 Erin Brockovich 2000.720p ][ 56 5.239891 Stargate-The Ark of Truth 720p ][ 57 6.004073 Leaving Las Vegas  720p ][ 58 4.944464 Incredibles, The 1080p ][ 62 1.311995 G.I.Joe The Rise of Cobra [2009] [720p] ][ 64 6.100061 The.Warlords.2007.720p.BluRay.DTS.x264-ESiR ][ 65 1.848927 Punisher 2   720p ][ 66 5.759800 Jumper    1080p ][ 70 4.510654 Doomsday 720p ][ 72 6.411729 SAW III   720p ][ 79 1.444821 The Matrix   1080p ][ 81 7.386294 Kung Fu Panda [2008]   720p ][ 82 8.131417 Charlie and the Chocolate Factory   720p ][ 83 7.210427 National Geographic - The Koran [720p] [Documenter] ][ 84 6.077602 21   1080p ][ 85 8.317092 12 Rounds 720p ][ 86 4.789126 Hitman    720p ][ 88 7.326222 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 91 9.302142 Bangkok Dangerous 720p ][ 93 7.309793 Pirates of Caribbean 1-2-3 Collection   1080p ][ 95 6.731524 Into The Blue 720P ][ 99 0.480487 One Missed Call   720p ][ 100 2.894432 Liar Liar    720p ][ 101 6.703355 The Secret of Moonarce [720p] ][ 104 6.991941 Core   720p ][ 105 2.985910 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC ][ 107 3.750715 The Notebook [2004] [1080p] ][ 111 3.670414 007.1967.You.Only.Live.Twice.720p.hdtv.x264-nbs ][ 114 3.753160 Terminator 2 Judgement Day Directors Cut [1991]   1080p ][ 118 7.655537 Alvin And The Chipmunks [720p] ][ 120 4.504557 Hancock    1080p ][ 123 6.170543 Babel.720p ][ 124 0.016130 X-Men 4 - Origins Wolverine (2009)  1080p ][ 125 3.012549 catch me if u can - 720p ][ 138 4.200550 Crank 2, High Voltage   1080p ][ 140 8.993602 The Proposal [2009] [720p] ][ 141 5.388408 Dr Dolite 3    720p ]
